ngraph/frontend/paddlepaddle/src/model.cpp

```diff
@@ -183,7 +183,7 @@ std::shared_ptr<TensorPlacePDPD> castToTensorPlace(const Place::Ptr& place) {
     } else if (auto out_port_place = std::dynamic_pointer_cast<OutPortPlacePDPD>(place)) {
         return out_port_place->getTargetTensorPDPD();
     }
-    PDPD_THROW("Cannot cast this Place to TensorPlacePDPD.");
+    PDPD_CHECK(false, "Cannot cast this Place to TensorPlacePDPD.");
```

Comment:
does it produce compilation warnings, like 'no return value' ?

Reply:
I didn't see any compilation warnings